MOAction/Windows/Config/ConfigWindow.Wizard.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using Dalamud.Interface;
+using Dalamud.Interface.Colors;
+using Dalamud.Interface.Components;
+using Dalamud.Interface.Utility.Raii;
+using ImGuiNET;
+using MOAction.Configuration;
+using Newtonsoft.Json;
+
+namespace MOAction.Windows.Config;
+
+public partial class ConfigWindow
+{
+    private int Wizard()
+    {
+        using var tabItem = ImRaii.TabItem("Wizard");
+        if (!tabItem.Success)
+            return 0;
+
+        ImGui.TextUnformatted("This window has some beginner friendly stacks for you to import.");
+        // TODO: don't hardcode import strings
+        //TODO maybe use the files in the folder below, this is that json, compacted & BASE64 encoded
+        if (ImGui.Button("Import Gapcloser basics"))
+        {
+            ImportStringToMouseOverActions("W3siQmFzZUlkIjoxNTUsIlN0YWNrIjpbeyJJdGVtMSI6IlVJIE1vdXNlb3ZlciIsIkl0ZW0yIjoxNTV9LHsiSXRlbTEiOiJGaWVsZCBNb3VzZW92ZXIiLCJJdGVtMiI6MTU1fSx7Ikl0ZW0xIjoiQ3Jvc3NoYWlyIiwiSXRlbTIiOjE1NX0seyJJdGVtMSI6IlRhcmdldCIsIkl0ZW0yIjoxNTV9XSwiTW9kaWZpZXIiOjAsIkpvYklkeCI6MjV9LHsiQmFzZUlkIjoyNTc2MiwiU3RhY2siOlt7Ikl0ZW0xIjoiVUkgTW91c2VvdmVyIiwiSXRlbTIiOjI1NzYyfSx7Ikl0ZW0xIjoiRmllbGQgTW91c2VvdmVyIiwiSXRlbTIiOjI1NzYyfSx7Ikl0ZW0xIjoiQ3Jvc3NoYWlyIiwiSXRlbTIiOjI1NzYyfSx7Ikl0ZW0xIjoiVGFyZ2V0IiwiSXRlbTIiOjI1NzYyfV0sIk1vZGlmaWVyIjowLCJKb2JJZHgiOjIwfSx7IkJhc2VJZCI6MjI2MiwiU3RhY2siOlt7Ikl0ZW0xIjoiVUkgTW91c2VvdmVyIiwiSXRlbTIiOjIyNjJ9LHsiSXRlbTEiOiJGaWVsZCBNb3VzZW92ZXIiLCJJdGVtMiI6MjI2Mn0seyJJdGVtMSI6IkNyb3NzaGFpciIsIkl0ZW0yIjoyMjYyfSx7Ikl0ZW0xIjoiVGFyZ2V0IiwiSXRlbTIiOjIyNjJ9XSwiTW9kaWZpZXIiOjAsIkpvYklkeCI6MzB9LHsiQmFzZUlkIjoyNDI5NSwiU3RhY2siOlt7Ikl0ZW0xIjoiVUkgTW91c2VvdmVyIiwiSXRlbTIiOjI0Mjk1fSx7Ikl0ZW0xIjoiRmllbGQgTW91c2VvdmVyIiwiSXRlbTIiOjI0Mjk1fSx7Ikl0ZW0xIjoiQ3Jvc3NoYWlyIiwiSXRlbTIiOjI0Mjk1fSx7Ikl0ZW0xIjoiVGFyZ2V0IiwiSXRlbTIiOjI0Mjk1fV0sIk1vZGlmaWVyIjowLCJKb2JJZHgiOjQwfSx7IkJhc2VJZCI6MzQ2NDYsIlN0YWNrIjpbeyJJdGVtMSI6IlVJIE1vdXNlb3ZlciIsIkl0ZW0yIjozNDY0Nn0seyJJdGVtMSI6IkZpZWxkIE1vdXNlb3ZlciIsIkl0ZW0yIjozNDY0Nn0seyJJdGVtMSI6IkNyb3NzaGFpciIsIkl0ZW0yIjozNDY0Nn0seyJJdGVtMSI6IlRhcmdldCIsIkl0ZW0yIjozNDY0Nn1dLCJNb2RpZmllciI6MCwiSm9iSWR4Ijo0MX1d");
```

Comment:
Just ship the .json files with the plugin and base64 encode them on plugin load :)
Can even EmbeddedResource them directly with the build .dll